codecs/move-gst-plugins.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+mkdir -p /app/lib/codecs/lib/gstreamer-1.0
+export GST_PLUGIN_SYSTEM_PATH=/app/lib/gstreamer-1.0
+
+for i in /app/lib/gstreamer-1.0/*.so; do
+  SUITE=`gst-inspect-1.0 $i | grep "Source module" | sed 's,Source module,,' | tr -d [:blank:]`
+  if [ x$SUITE != x'gst-plugins-bad' ] &&
+     [ x$SUITE != x'gst-plugins-ugly' ] &&
+     [ x$SUITE != x'gst-libav' ]; then
```

Comment:
Is this sense the right way round – only move `$i` if it is **not** in -bad / -ugly / -libav?

Reply:
Definitely the wrong way around:
```
$ ls /app/lib/gstreamer-1.0/libgstlibav.so 
/app/lib/gstreamer-1.0/libgstlibav.so
```